govcd/provider_vdc_test.go

```diff
@@ -42,7 +43,10 @@ func (vcd *TestVCD) Test_GetProviderVdc(check *C) {
 		check.Assert(*providerVdc.ProviderVdc.IsEnabled, Equals, true)
 		check.Assert(providerVdc.ProviderVdc.ComputeCapacity, NotNil)
 		check.Assert(providerVdc.ProviderVdc.Status, Equals, 1)
-		check.Assert(len(providerVdc.ProviderVdc.NetworkPoolReferences.NetworkPoolReference), Equals, 1)
+		// This test may fail when the VCD has more than one network pool depending on the same NSX-T manager
```

Comment:
The below note that you have in PR's description is very useful. It seems useful to have somewhere in the code as well?

> Note: you need to have at least one free resource pool to test provider VDC creation, and at least two of them to test update.

Reply:
The two annotations are not related, but I can add the information about the resource pools in the test header

Reply:
Thanks!

p.s. And, yes, I just didn't find a better place where to put the note :) As GitHub allows adding review notes only on the changed lines.